config/application.php

```diff
@@ -81,16 +81,15 @@
 Config::define('DB_HOST', env('DB_HOST') ?: 'localhost');
 Config::define('DB_CHARSET', 'utf8mb4');
 Config::define('DB_COLLATE', '');
-$table_prefix = env('DB_PREFIX') ?: 'wp_';
 
-if (env('DATABASE_URL')) {
-    $dsn = (object) parse_url(env('DATABASE_URL'));
+// If the request origin is from a test suite, use the test database.
+$is_test_request = ( isset($_SERVER['HTTP_X_TEST_REQUEST']) && $_SERVER['HTTP_X_TEST_REQUEST'] )
+|| ( isset($_SERVER['HTTP_USER_AGENT']) && $_SERVER['HTTP_USER_AGENT'] === 'wp-browser' )
+|| getenv('WPBROWSER_HOST_REQUEST');
+
+// Set the table prefix based on the request origin.
+$table_prefix =  $is_test_request ? 'test_' : (env('DB_PREFIX') ?: 'wp_');
 
-    Config::define('DB_NAME', substr($dsn->path, 1));
-    Config::define('DB_USER', $dsn->user);
-    Config::define('DB_PASSWORD', $dsn->pass ?? null);
-    Config::define('DB_HOST', isset($dsn->port) ? "{$dsn->host}:{$dsn->port}" : $dsn->host);
-}
```

Comment:
I can't see a replacement here yet, a URL has been defined in the `.env.example` file. 
Is this important?

Reply:
I've checked it looks like we're not using `DATABASE_URL` for local or deployments.

I think it would be for if we use a connection string like `mysql://<dbhost>:<dbport>/<dbname>`.

My thoughts are we probably won't start using `DATABASE_URL` so let's remove it from the codebase.